gluten-ut/spark35/src/test/resources/backends-velox/sql-tests/inputs/linear-regression.sql

```diff
@@ -46,7 +46,7 @@ SELECT k, regr_slope(y, x) FROM testRegression GROUP BY k;
 SELECT k, regr_slope(y, x) FROM testRegression WHERE x IS NOT NULL AND y IS NOT NULL GROUP BY k;
 
 -- SPARK-37623: Support ANSI Aggregate Function: regr_intercept
-SELECT regr_intercept(y, x) FROM testRegression;
-SELECT regr_intercept(y, x) FROM testRegression WHERE x IS NOT NULL AND y IS NOT NULL;
-SELECT k, regr_intercept(y, x) FROM testRegression GROUP BY k;
-SELECT k, regr_intercept(y, x) FROM testRegression WHERE x IS NOT NULL AND y IS NOT NULL GROUP BY k;
+-- SELECT regr_intercept(y, x) FROM testRegression;
```

Comment:
related with:
6be6176a3 by Jimmy Lu, fix: Support float to double schema evolution (13317)

Reply:
@zhouyuan, could you submit an issue to track?

Reply:
I fired one here:
https://github.com/apache/incubator-gluten/issues/9633